core/trino-main/src/main/java/io/trino/server/security/oauth2/OAuth2Config.java

```diff
@@ -45,6 +50,7 @@
     private String tokenUrl;
     private String jwksUrl;
     private Optional<String> userinfoUrl = Optional.empty();
+    private Optional<String> userinfoRequestMethod = Optional.empty();
```

Comment:
Why is this configurable? Does oauth2 specify both methods for this type of API?

Reply:
https://openid.net/specs/openid-connect-core-1_0.html#UserInfoRequest 

https://github.com/trinodb/trino/issues/11013